### PR TITLE
New indicator status page revisions from approval process

### PR DESCRIPTION
**Pull request by krivard**
**Prerequisites**:

- [x] Unless it is a hotfix it should be merged against the `dev` branch
- [x] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [x] Build is successful
- [x] Code is cleaned up and formatted

### CODE REVIEW

File: src/modes/indicator-status/IndicatorSignal.svelte [hunk 1]
```diff
@@ -131,7 +140,7 @@
           <div>
             <KPI text={formatDateISO(metaData ? metaData.maxIssue : null)} />
           </div>
-          <div class="sub">Latest Issue</div>
+          <div class="sub" title="Date the most recent update was published">Latest Issue</div>
```
Comment (krivard) [inline src/modes/indicator-status/IndicatorSignal.svelte:143]:
```suggestion
          <div class="sub" title="Date the most recent update was published by Delphi">Latest Issue</div>
```

File: src/modes/indicator-status/IndicatorStatusTable.svelte [hunk 2]
```diff
@@ -71,12 +71,14 @@
     </tr>
     <tr>
       <th />
-      <th class="mobile-th uk-text-right">Latest Issue</th>
-      <th class="mobile-th uk-text-right">Latest Data Available</th>
+      <th class="mobile-th uk-text-right" title="Date the most recent update was published">Latest Issue</th>
```
Comment (krivard) [inline src/modes/indicator-status/IndicatorStatusTable.svelte:74]:
```suggestion
      <th class="mobile-th uk-text-right" title="Date the most recent update was published by Delphi">Latest Issue</th>
```

**Comment by krivard** [comment]
All text approved by Roni.

**Review by sgratzl — APPROVED** [review]
👍

**Pull request merged**